pennylane/gradients/parameter_shift.py

```diff
@@ -215,17 +260,16 @@ def expval_param_shift(tape, argnum=None, shifts=None, gradient_recipes=None, f0
 
     gradient_tapes = []
     # Each entry for gradient_data will be a tuple with content
-    # (num_tapes, coeffs, fn, unshifted_coeff)
+    # (num_tapes, coeffs, fn, unshifted_coeff, batch_size)
     gradient_data = []

```

Comment:
Minor: even originally, was contemplating: would it be worth having this as a namedtuple? I figured eventually that readers later in the file would scroll up to this point and see the same thing.

Reply:
Yep, no strong opinion here either. I think the unpacking in the `for` loop makes the current approach quite nice still. I think both ways have pros and cons. Happy for this to be left as is. :+1: